app/src/main/res/layout/mark_chapters_completed_fragment.xml

```diff
@@ -112,7 +112,7 @@
         android:id="@+id/mark_chapters_completed_recycler_view"
```

Comment:
In line `96` color `color_def_white ` needs to be fixed too.

Reply:
> In line `96` color `color_def_white ` needs to be fixed too.

Done